src/themes/default/elements/icon.overrides

```diff
@@ -891,3 +917,13 @@ i.icon.commenting:before { content: "\f27a"; }
 i.icon.commenting.outline:before { content: "\f27b"; }
 i.icon.edge:before { content: "\f282"; }
 i.icon.ms.edge:before { content: "\f282"; }
+i.icon.wordpress.beginner:before { content: "\f297"; }
+i.icon.wordpress.forms:before { content: "\f298"; }
+i.icon.envira:before { content: "\f299"; }
+i.icon.question.circle.outline:before { content: "\f29c"; }
```

Comment:
`question circle outline` is an alias for `help circle outline`. Pretty straightforward.